src/kubernetes-tool/templates/splash_screen.vue

```diff
@@ -33,15 +33,30 @@ limitations under the License.
                 <div class="input-container">
                     <label for="helmInput" class="hidden">{{ i18n.templates.splashScreen.helmTitle }}</label>
                     <i class="fas fa-dharmachakra"></i>
-                    <input id="helmInput"
-                           v-model="helmId"
-                           class="input"
-                           type="text"
-                           :placeholder="i18n.templates.splashScreen.helmTitle"
-                    />
-                    <button id="submitHelm" class="button is-primary" :click="submitHelm">
-                        {{ i18n.templates.splashScreen.submit }}
-                    </button>
+                    <vue-autosuggest
+                        v-model="helmId"
+                        :suggestions="helmSuggestions"
+                        :input-props="{id: 'helmInput', class: 'input', type: 'text', placeholder: i18n.templates.splashScreen.helmTitle, style: {width: '80%', textIndent: '30px'}, readonly: readonly}"
+                        :style="{width: '100%'}"
+                        @input="inputChange"
+                        @selected="inputSelect"
+                    >
+                        <template slot-scope="{suggestion}">
+                            <div :style="{backgroundColor: 'white'}">
+                                <p :style="{margin: '0', textAlign: 'left', fontSize: '17px'}">
+                                    <a href="javascript:void(0)" tabindex="0" @click="inputSelect(suggestion)">
+                                        {{ suggestion.item }}
+                                    </a>
+                                </p>
+                            </div>
+                        </template>
+
+                        <template slot="after-input">
+                            <button id="submitHelm" class="button is-primary" :click="submitHelm">
```

Comment:
What's the different between `@click` & `:click`?
We seem to use `@click` normally?

Reply:
I *think* that it's something relating to events, just tested this, `@click` seems to do the default event (then I assume emit, but it never actually gets to that stage)